keyboards/wsk/jerkin/config.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    qmkbuilder
```

Comment:
You want to fill out these defaults with sane values, just guessing with these recommendations.
```suggestion
#define MANUFACTURER    wsk
```